jax_rocm_plugin/build/rocm/tools/get_rocm.py

```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env python3
+"""Script for installing ROCm from various places
 
-# Copyright 2024 The JAX Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
```

Comment:
Copyright detectors usually need this to be the first thing, using the # comments, and then you can put the module level docstring afterwards.